mtp_api/apps/transaction/permissions.py

```diff
@@ -1,11 +1,29 @@
 from rest_framework.permissions import BasePermission
 
+from core.permissions import ActionsBasedPermissions
+
+
 class IsOwner(BasePermission):
 
     def has_permission(self, request, view):
         return str(request.user.id) == view.kwargs.get('user_id')
 
+
 class IsOwnPrison(BasePermission):
 
     def has_permission(self, request, view):
         return request.user.prisonusermapping.prisons.filter(pk=view.kwargs.get('prison_id')).exists()
+
+
+class TransactionPermissions(ActionsBasedPermissions):
+    actions_perms_map = {
+        'create': ['%(app_label)s.add_%(model_name)s'],
+        'list': ['%(app_label)s.view_%(model_name)s'],
+        'retrieve': ['%(app_label)s.view_%(model_name)s'],
+        'update': ['%(app_label)s.change_%(model_name)s'],
+        'partial_update': ['%(app_label)s.change_%(model_name)s'],
+        'destroy': ['%(app_label)s.delete_%(model_name)s'],
+        'take': ['%(app_label)s.take_%(model_name)s'],
+        'release': ['%(app_label)s.release_%(model_name)s'],
+        'patch_credited': ['%(app_label)s.patch_credited_%(model_name)s']
+    }
```

Comment:
Didn't know that you guys would have hated overriding the dict so much :-|
I don't like so much the other options but I'll go for whatever you guys like, it's a small thing so I'm not picky.

@SteveMarshall are you happy with 

```
class TransactionPermissions(ActionsBasedPermissions):
    action_perms_map = ActionBasedPermissions.action_perms_map.copy()
    action_perms_map.update({...})
```
?